Core/PlexAPI/PlexAuth.cs

```diff
@@ -1,107 +1,100 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net.Http;
-using System.Text;
-using System.Text.Json;
-using System.Threading.Tasks;
+﻿namespace PlexBot.Core.PlexAPI;
 
-namespace PlexBot.Core.PlexAPI
+public class PlexAuth
 {
-    public class PlexAuth
+    readonly string clientIdentifierKey = Environment.GetEnvironmentVariable("CLIENT_IDENTIFIER_KEY") ?? "";
+    readonly string plexAppName = Environment.GetEnvironmentVariable("PLEX_APP_NAME") ?? "";
+    readonly HttpClient httpClient = new();
+
+    //TODO: Come back to this later and make it work for ILogger in the Microsoft.Extensions.Logging namespace
```

Comment:
This class in currently unused and will be turned into a plugin. No need to spend anymore time on this.